endpoints/openrtb2/amp_auction.go

```diff
@@ -146,15 +148,16 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 	w.Header().Set("Access-Control-Expose-Headers", "AMP-Access-Control-Allow-Source-Origin")
 	w.Header().Set("X-Prebid", version.BuildXPrebidHeader(version.Ver))
 
+	_, rejectErr := deps.hookExecutor.ExecuteEntrypointStage(r, nil)
```

Comment:
The `entrypoint` stage is executed at both endpoints, "auction" and "amp", however, the `amp` request does not have HTTP body, thus we do not pass it in the parameters to `ExecuteEntrypointStage ` and ignore its return value.
I have also updated the code, instead of passing `nil`, I pass a package-scoped variable called  `nilBody`.